src/Centreon/Application/Controller/MonitoringResourceController.php

```diff
@@ -238,6 +238,9 @@ public function list(
 
             // add shortcuts
             $this->provideLinks($resource, $contact);
+
+            // replace macros in external links
+            $this->resource->replaceMacrosInExternalLinks($resource);
```

Comment:
It's not controller related, this method should be called directly from domain
ie: if you add a CLI controller, you don't have to call this method. Domain should provides directly expected data